src/pages/contract/QueryContract.tsx

```diff
@@ -61,25 +50,33 @@ export function QueryContract({ contractAddress }: Props): JSX.Element {
       <ul className="list-group list-group-flush">
         <li className="list-group-item d-flex align-items-baseline">
           <span title="The contract query input">Query contract:</span>
-          <input
-            className="ml-3 flex-grow-1"
-            value={queryInput}
-            onChange={(event) => setQueryInput(event.target.value)}
+        </li>
+        <li className="list-group-item d-flex align-items-baseline">
+          <JSONInput
```

Comment:
Can we configure this component to only accept JSON, not all JavaScript?

Reply:
Answer seems to be no. See e.g. https://github.com/AndrewRedican/react-json-editor-ajrm/issues/120